packages/esm-patient-registration-app/translations/en.json

```diff
@@ -29,32 +23,23 @@
   "estimatedMonthsLabelText": "Estimated Months",
   "estimatedYearsLabelText": "Estimated Years",
   "familyNameLabelText": "Family Name",
-  "familyNameRequired": "Family name is required",
```

Comment:
Are you sure we don't need these translations anymore?

Reply:
These translations got removed in the previous commit, after I removed the commented translations from address field.
Let me just put these all back.

Thank you for noticing it.